README.md

```diff
@@ -284,30 +284,36 @@ Obtain overlap-aware speaker embeddings from a microphone stream:
 ```python
 import rx.operators as ops
 import diart.operators as dops
-from diart.sources import MicrophoneAudioSource
+from diart.sources import MicrophoneAudioSource #, FileAudioSource
 from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding
 
-segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
-embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")
+segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation", use_hf_token="")
+embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding", use_hf_token="")
```

Comment:
```suggestion
segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")
```

I prefer that the unused parameters are left to their default value when there is one. In this case, the default will look for the token from the huggingface cli login